### DISCUSSION TITLE
Announcing Sepolia and deprecating Goerli

### DISCUSSION THREAD

**Discussion by agolajko** [zkSync-Community-Hub/zksync-developers#228] [Announcements]
# **TLDR;** The zkSync Sepolia Testnet is now live! We will be sunsetting zkSync Era Goerli on March 31st 2024. 🔊🚨

![rsz_cat_sepolia](https://github.com/zkSync-Community-Hub/zkync-developers/assets/57454127/001a418e-6107-4343-9fc9-9ab36be0a271)

### ⏱️ Background & Motivation

Goerli testnet was [deprecated as of January 2023](https://github.com/eth-clients/goerli#readme), and long-term support stops on March 31st, 2024. This indicates that all projects relying on Goerli testnet will have to migrate to the new zkSync Sepolia Testnet. This new network will be our primary testing ground for the coming years so you can expect future updates to appear here.

### 🔀 Will there be data and services migration?
> ⚠️⚠️⚠️ **The state of the new Sepolia testnet will be entirely blank with no data being migrated over from Goerli.** ⚠️⚠️⚠️

We will be in touch with our ecosystem partners to support their migration and answer any upcoming questions.

### ✅ Todos for devs

Please update your RPCs/links to:

- zkSync Sepolia RPC: https://sepolia.era.zksync.dev/
- zkSync Sepolia Block Explorer: https://sepolia.explorer.zksync.io/
- Sepolia Faucets:
  - https://faucet.quicknode.com/ethereum/sepolia
  - https://sepoliafaucet.com/
- zkSync Sepolia Bridge: https://portal.zksync.io/bridge/?network=sepolia

### 👉🏻  zkSync Lite
 ✔️  **zkSync Lite** is migrated to Sepolia
⚠️ The new Sepolia testnet will be entirely blank, with no data being migrated from Goerli

**Comment by JackHamer09** [comment]
Might be useful for someone:

`zkSync Sepolia Testnet`:
```js
{
  id: 300,
  name: "zkSync Sepolia Testnet",
  rpcUrl: "https://sepolia.era.zksync.dev",
  blockExplorerUrl: "https://sepolia.explorer.zksync.io",

  // other info
  blockExplorerApi: "https://block-explorer-api.sepolia.zksync.dev",
  contractVerificationUrl: "https://explorer.sepolia.era.zksync.dev/contract_verification",
  bridgeUrl: "https://portal.txsync.io/bridge/?network=era-sepolia",
  walletUrl: "https://portal.txsync.io/?network=era-sepolia",
}
```

`Ethereum Sepolia Testnet` (L1):
```js
{
  id: 11155111,
  name: "Ethereum Sepolia Testnet",
  rpcUrl: "https://rpc.ankr.com/eth_sepolia",
  blockExplorerUrl: "https://sepolia.etherscan.io",
}
```

**Reply by arnolwijaya77** [reply]
Is this mean that bridging from goerli zksync to goerli ethereum wont work as usual ?

**Reply by uF4No** [reply]
it will work the same for a while but Goerli is being deprecated so I suggest you to migrate to both Etherem Sepolia and zkSync Sepolia.

**Reply by 0xiliyan** [reply]
At one point only Sepolia ETH -> Sepolia zkSync and vise versa will be used as the testnet as Goerli is being deprecated across all projects on Ethereum

**Reply by DiazEtoz** [reply]
is the bridge from Goerli still good to bridge out some left over eth?

**Reply by DayOfShah** [reply]
whats up with those cartoon animals?

**Reply by MildSandy** [reply]
Please be sure to migrate on Sepolio Testnet, Goerli will be closed soon

**Reply by NPCoc** [reply]
good job

**Comment by jayjose1** [comment]
nice update

**Reply by enerelt0418** [reply]
good job

**Comment by Tuneezi** [comment]
Amazing

**Comment by jayjose1** [comment]
thanks for update
    On Tuesday, December 12, 2023 at 07:35:27 PM GMT+8, agolajko ***@***.***> wrote:  
 
 


We now have the External Node ready and available for partners and the general public! Please update your config files to the following, you can also download the backup or sync from genesis with an empty database

**Comment by jayjose1** [comment]
it is node- for zksync! need to run node via vps or what


Sent from Yahoo Mail for iPhone


On Tuesday, December 12, 2023, 8:27 AM, Tuneezi ***@***.***> wrote:




Amazing

**Comment by vasinl124** [comment]
What about bridging Goerli to Sepolia? possible?

**Comment by cryptoliang** [comment]
Does The Graph support zkSync Sepolia?

**Reply by uF4No** [reply]
Yes it does

**Comment by samui68400** [comment]
Ok so why did you guys choose Sepolia and not holesky?

**Comment by jayjose1** [comment]
i forgot use holesky

**Reply by samui68400** [reply]
that's what I thought

**Comment by arnolwijaya77** [comment]
Is that mean bridging from zksync goerli to ethereum goerli wont work as usual ?

**Comment by Vijay2031** [comment]
Yes it's working fine and working is good

**Comment by jayjose1** [comment]
wow working good zksync best chain zk proof layer2

**Comment by Vijay2031** [comment]
Yes it's true

**Comment by dzhamilrustamov** [comment]
Good

**Comment by dzhamilrustamov** [comment]
😉

**Comment by montax5** [comment]
Its better to add Holesky as well , cos getting that from faucet is not a problem for many

**Comment by enerelt0418** [comment]
> # **TLDR;** The zkSync Sepolia Testnet is now live! We will be sunsetting zkSync Era Goerli in 2024.
> ![rsz_cat_sepolia](https://private-user-images.githubusercontent.com/57454127/288473174-001a418e-6107-4343-9fc9-9ab36be0a271.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgwNTMxMjYsIm5iZiI6MTcwODA1MjgyNiwicGF0aCI6Ii81NzQ1NDEyNy8yODg0NzMxNzQtMDAxYTQxOGUtNjEwNy00MzQzLTlmYzktOWFiMzZiZTBhMjcxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE2VDAzMDcwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRjOTMyNjVmNmIzMzMyOGVkM2RkNjlkZmFlMzE2ZWY5MWYyM2Y2NzVhZjI4ZmE1M2YyNjhkZTBmY2M0M2EwODEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.BBtreEqcNViX5Rh_LvxI4o-DE-CXAfdVKe866JJ6Jps)
> 
> ### ⏱️ Background & Motivation
> Goerli testnet was [deprecated as of January 2023](https://github.com/eth-clients/goerli#readme) and long-term support stops in early 2024. zkSync will also end support starting in early 2024, meaning all projects relying on it will have to migrate to the new zkSync Sepolia Testnet. This new network will be our primary testing ground for the coming years so you can expect future updates to appear here.
> 
> ### 🔀 Will there be data and services migration?
> > ⚠️⚠️⚠️ **The state of the new Sepolia testnet will be entirely blank with no data being migrated over from Goerli.** ⚠️⚠️⚠️
> 
> We will be in touch with our ecosystem partners to support their migration and answer any upcoming questions.
> 
> ### ✅ Todos for devs
> Please update your RPCs/links to:
> 
> * zkSync Sepolia RPC: https://sepolia.era.zksync.dev/
> * zkSync Sepolia Block Explorer: https://sepolia.explorer.zksync.io/
> * Sepolia Faucets:
>   
>   * https://faucet.quicknode.com/ethereum/sepolia
>   * https://sepoliafaucet.com/
> * zkSync Sepolia Bridge: https://portal.zksync.io/bridge/?network=sepolia

I will try this one

**Reply by evmparser** [reply]
Yes it's working fine

**Comment by m1s3ryyy** [comment]
nice update

**Comment by therntibmaph** [comment]
Attention developers: zkSync Sepolia Testnet is now live! Please update your RPC endpoints, block explorer links, and faucet URLs accordingly. Remember, there will be no data migration from zkSync Era Goerli, so be prepared to migrate your projects to Sepolia. Stay tuned for future updates and reach out if you have any questions! #zkSync #SepoliaTestnet 🚀🔗

**Comment by stennettmeisha** [comment]
zkSync Sepolia Testnet is live! Prepare for the transition from zkSync Era Goerli to zkSync Sepolia by updating your RPC endpoints, exploring the new block explorer, and utilizing the Sepolia faucets and bridge. Happy testing! 🛠️🔗

**Comment by yas3i3** [comment]
IT WOULD BE A NICE CONTRIBUTION.

**Comment by jayjose1** [comment]
nice Zksync tean

Sent from Yahoo Mail for iPhone


On Sunday, March 3, 2024, 3:23 AM, yas3i3 ***@***.***> wrote:




IT WOULD BE A NICE CONTRIBUTION.

**Comment by sundaramjacob** [comment]
zkSync Sepolia Testnet is live! 

 Transition from zkSync Era Goerli to Sepolia Testnet:

zkSync Era Goerli support ending in 2024
Migration to zkSync Sepolia Testnet required
No data migration from Goerli to Sepolia Testnet
  Developer Todos:

Update RPCs/links to Sepolia Testnet
Utilize new Sepolia RPC and Block Explorer
Access Sepolia Faucets for testing
Utilize zkSync Sepolia Bridge for network interaction
Let's migrate and continue building on zkSync Sepolia Testnet!

**Comment by jennifercerso** [comment]
zkSync has launched the Sepolia Testnet as a replacement for the deprecated Goerli testnet. The new Sepolia testnet will be the primary testing ground for zkSync going forward and will require developers to update their RPCs/links. However, there will be no data migration from the Goerli testnet to the Sepolia testnet. Developers are advised to update their links to the zkSync Sepolia RPC, block explorer, and bridge.

**Comment by yas3i3** [comment]
Good 👍

**Comment by Mfusa** [comment]
zksync it's number one

**Comment by marmotte91** [comment]
Of course it is


Envoyé à partir de Courrier<https://go.microsoft.com/fwlink/?LinkId=550986> pour Windows

________________________________
De : Mfusa ***@***.***>
Envoyé : Wednesday, March 20, 2024 7:47:45 PM
À : zkSync-Community-Hub/zksync-developers ***@***.***>
Cc : marmotte91 ***@***.***>; Manual ***@***.***>
Objet : Re: [zkSync-Community-Hub/zksync-developers] Announcing Sepolia and deprecating Goerli (Discussion #228)


zksync it's number one

**Comment by andrewtotta** [comment]
The zkSync team has announced the launch of the Sepolia Testnet, which will be the primary testing ground for future updates. The Goerli testnet will be deprecated in March 2024, and all projects relying on it will need to migrate to Sepolia. There will be no data migration from Goerli to Sepolia, and developers are advised to update their RPCs/links and use the new Sepolia Faucets and Bridge. The zkSync Lite has already been migrated to Sepolia.

**Comment by AslanHz** [comment]
Interesting sepolia is much smother tan goerli

**Comment by yehia67** [comment]
Thats great! and I would recommend https://learnweb3.io/faucets/zksync_sepolia/ for faucets too!

**Comment by saniaus** [comment]
how get eth testnet ser?

**Reply by MildSandy** [reply]
L2faucet.com

Or if you have sepolio eth then bridge it

**Comment by IronSide4488** [comment]
Normally

Вс, 9.06.2024 в 22:07, Sania ***@***.***>:

**Comment by DayOfShah** [comment]
from the faucet

**Comment by rchak007** [comment]
hi i am testing swapping WETH to USDC etc on ZkSync sepolia. Any swap router addresses we can use that has liquidity and we can swap for testing. Thx.
